### ISSUE TITLE
Regression: TypeError: path.charCodeAt is not a function

### ISSUE THREAD

**Issue by johncrim** [oddbird/true#267]
Upgrading from `"sass-true": "^6.1.0"` to `"sass-true": "^7.0.0"`, my sass tests fail to run. I get:

```
 FAIL  libs/style/test/sass-tests.js
  ● Test suite failed to run
                                                                                                                                                                                                                                                                 
    TypeError: path.charCodeAt is not a function

      21 | describe('Sass', () => {
      22 |   sassTestFiles.forEach(file => {
    > 23 |     sassTrue.runSass(
         |              ^
      24 |       {
      25 |         file,
      26 |         includePaths

      at WindowsStyle.rootLength$2$withDrive (../../../node_modules/sass/sass.dart.js:41344:16)
      at WindowsStyle.rootLength$1 (../../../node_modules/sass/sass.dart.js:41369:19)
      at Context._needsNormalization$1 (../../../node_modules/sass/sass.dart.js:40631:19)
      at Context.normalize$1 (../../../node_modules/sass/sass.dart.js:40622:17)
      at Object.compile (../../../node_modules/sass/sass.dart.js:22165:47)
      at StaticClosure.compile0 (../../../node_modules/sass/sass.dart.js:21752:20)
      at Object.Primitives_applyFunction (../../../node_modules/sass/sass.dart.js:2001:30)
      at Object.Function_apply (../../../node_modules/sass/sass.dart.js:6267:16)
      at _callDartFunctionFast (../../../node_modules/sass/sass.dart.js:8128:16)
      at sass.compile (../../../node_modules/sass/sass.dart.js:8106:18)
      at Object.runSass (../../../node_modules/sass-true/src/index.ts:77:21)
      at runSass (sass-tests.js:23:14)
          at Array.forEach (<anonymous>)
      at forEach (sass-tests.js:22:17)
      at Object.describe (sass-tests.js:21:1)

Test Suites: 1 failed, 1 total                                                                                                                                                                                                                                   
Tests:       0 total                                                                                                                                                                                                                                             
Snapshots:   0 total
Time:        6.777 s, estimated 11 s
```

I'm using:
"jest": "^29.7.0",
"sass": "~1.69.6"

And this is on windows - I haven't tested on our linux build agents yet. This issue has occurred since sass-true 7.0.0 was first released, but I didn't report the issue earlier. I have not had any other issues with `sass.dart`.

My jest test file looks pretty standard:
```js
// sass-tests.js
/**
 * This file runs SASS tests in jest:
 * yarn sass:test
 */

const path = require('path');
const fs = require('fs');
const sass = require('sass');
const sassTrue = require('sass-true');
const glob = require('glob');

const includePaths = [
  __dirname,
  path.join(__dirname, '../src')
];

const sassTestFiles = glob.sync(
  path.resolve(process.cwd(), './**/*.spec.scss')
)

describe('Sass', () => {
  sassTestFiles.forEach(file => {
    sassTrue.runSass(
      {
        file,
        includePaths
      },
      {
        sass,
        describe,
        it
      });
  });

});
```

If I revert to sass-true 6.1.0 things continue to work as expected. 

The 2 fns at the top of the call stack are:

![image](https://github.com/oddbird/true/assets/3312127/0f3d7db4-0b6d-4b46-abd8-592871768f1a)

Lmk if I can provide more info to help diagnose. If the problem isn't obvious, I can create a simple repro repo.

**Comment by jgerigmeyer** [comment]
@johncrim Hm, interesting. A simple repro repo might be useful if you're willing to provide that.

True v7 included [breaking changes](https://github.com/oddbird/true/releases/tag/v7.0.0) to the arguments that `runSass` accepts, and in your example it looks like you're still using the pattern from v6. Can you confirm that?

**Comment by johncrim** [comment]
@jgerigmeyer - thank you for the response! The problem was indeed that I hadn't changed the way I called `sassTrue.runSass()` - I had just updated without looking at the release notes.

It might be a good idea to validate the arg types given the change in signature (and given that the method name is the same as before). 

I'm going to close this as "user error".

Thank you for the guidance, and your work on this valuable project.
